proof/refine/ARM/TcbAcc_R.thy

```diff
@@ -2789,7 +2789,7 @@ lemma threadSet_queued_sch_act_wf[wp]:
   done
 
 lemma tcbSchedEnqueue_pred_tcb_at'[wp]:
-  "\<lbrace>\<lambda>s. pred_tcb_at' proj P' t' s \<rbrace> tcbSchedEnqueue t \<lbrace>\<lambda>_ s. pred_tcb_at' proj P' t' s\<rbrace>"
+  "\<lbrace>\<lambda>s. P (pred_tcb_at' proj P' t' s) \<rbrace> tcbSchedEnqueue t \<lbrace>\<lambda>_ s. P (pred_tcb_at' proj P' t' s)\<rbrace>"
```

Comment:
Should this use the invariant syntax?

Reply:
I know that it is better to do that, but that’s more for when you are writing a new lemma from scratch, I’d think? I can of course change this, too, but I am not going so strongly in the clean up everything I touch mode, particularly for this one...

Reply:
You're right, it's not a huge issue. I tend to try to do them, but I also don't do all of them. Up to you.